packages/api/amplify_api/test/dart_graphql_test.dart

```diff
@@ -146,8 +160,9 @@ class MockAmplifyAPI extends AmplifyAPIDart {
   });
 
   @override
-  WebSocketConnection getWebSocketConnection({String? apiName}) =>
-      MockWebSocketConnection(testApiKeyConfig, getTestAuthProviderRepo());
+  WebSocketConnection getWebSocketConnection({String? apiName}) {
+    return mockWebSocketConnection;
```

Comment:
Should it be the same for every test?

Reply:
Yeah, I believe this is okay given the tests implementations.